src/components/VContentPage.vue

```diff
@@ -0,0 +1,9 @@
+<template>
+  <div class="pt-5 md:pt-10" dir="ltr">
+    <div
+      class="px-6 lg:px-0 mb-10 lg:mb-30 md:max-w-4xl xl:max-w-5xl prose prose-sm md:prose-base mx-auto max-w-none prose-a:text-pink prose-headings:font-bold lg:prose-headings:text-3xl lg:prose-h1:text-6xl"
```

Comment:
Can we set the width a little smaller? Something closer to the '70 to 80 characters per line (about 8 to 10 words in English)' that is usually suggested for comfortable reading?

Reply:
It's much smaller than the current width of these pages, but maybe we could go even smaller. @panchovm could you look at this PR this week?